server/container_create.go

```diff
@@ -597,6 +578,9 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 	}
 
 	containerImageConfig := containerInfo.Config
+	if containerImageConfig == nil {
+		return nil, fmt.Errorf("empty image config for %s", image)
+	}
```

Comment:
so, this is an actual bug fix as w/o this check here we panic here https://github.com/kubernetes-incubator/cri-o/pull/572/files#diff-dba9598e19f7903533dc9add98d410d2R585